derive_builder/src/options/mod.rs

```diff
@@ -198,6 +206,9 @@ impl<Mode> OptionsBuilder<Mode> where
                 // setter implicitly enabled
                 self.setter_enabled(true)
             },
+            "try_setter" => {
+                self.try_setter_enabled(true)
```

Comment:
Or wait - let me think about it again. Initially I thought you were implementing an API like `#[builder(setter(try))]` as in [this comment](https://github.com/colin-kiegel/rust-derive-builder/issues/25#issuecomment-293044392).